Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.cs

```diff
@@ -104,8 +106,19 @@ private void CycleBlades()
             foreach (var blade in Items.OfType<BladeItem>())
             {
                 if (blade.IsOpen)
-                {
                     ActiveBlades.Add(blade);
+            }
+
+            // For now we skip this feature when blade mode is set to fullscreen
+            if (AutoCollapseCountThreshold > 0 && BladeMode != BladeMode.Fullscreen && ActiveBlades.Any())
+            {
+                var openBlades = ActiveBlades.Where(item => item.TitleBarVisibility == Visibility.Visible).ToList();
```

Comment:
My understanding was that the TitleBarVisibility property was used to pin blades to the screen.
I want to filter the pinned once out, they are not touched by the auto resize mechanism.
@skendrot Is there another property that tells me this? I don't seem to find it...